src/post/post.repository.knex.ts

```diff
@@ -947,4 +948,26 @@ export class KnexPostRepository {
 
         return result !== undefined;
     }
+
+    async getMentionsForPost(post: Post): Promise<Mention[]> {
+        const rows = await this.db('mentions')
+            .select(
+                'mentions.post_id',
+                'accounts.username as account_username',
+                'accounts.domain as account_domain',
+                'accounts.ap_id as account_apid',
+            )
+            .join('accounts', 'mentions.account_id', 'accounts.id')
+            .where({
+                post_id: post.id,
+            });
+
+        return rows.map(
+            (row) =>
+                new Mention({
+                    name: `@${row.account_username}@${row.account_domain}`,
+                    href: new URL(row.account_apid),
+                }),
+        );
```

Comment:
Wonder if we should be mapping to Fedify's Mentions directly in PostRepository here 🤔 

Think I would have expected us to return an array of the internal `Mention` interface here, and then map that to its Fediverse equivalent in Fediverse Bridge?

Reply:
Removed this method as now we have mentions added in post.